test/asserts.js

```diff
@@ -0,0 +1,10 @@
+import assert from "assert";
+
+export function assertInDelta(actual, expected, delta) {
+  assert(expected - delta <= actual && actual <= expected + delta, {
+    message: "should be in delta",
+    operator: "inDelta",
+    actual: actual,
+    expected: [expected - delta, expected + delta]
+  });
```

Comment:
I’m pretty sure that Node’s assert only takes a *value* and an optional *message*. This looks like Tape’s API?

https://nodejs.org/api/assert.html#assert_assert_value_message